lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -220,6 +225,8 @@ def test_get(self):
         data = response.data
         courses = data['results']
 
+        print courses
```

Comment:
lol, thanks for the catch. I'm getting a little sloppy trying to chuck these out.